app/javascript/tapp/components/assistant.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import { ListGroup, ListGroupItem } from 'react-bootstrap';
+
+class Assistant extends React.Component {
+    render() {
```

Comment:
assistant only needs a list of sessions and the ability to download a csv file containing the assignments for that session.